perfmetrics/scripts/hns_rename_folders_metrics/renaming_benchmark.py

```diff
@@ -39,6 +43,76 @@
 log = logging.getLogger()
 
 
+def _upload_to_gsheet(worksheet, data, spreadsheet_id) -> (int):
+  """
+  Writes rename results to Google Spreadsheets.
+  Args:
+    worksheet (str): Google sheet name to which results will be uploaded.
+    data (list): metrics to be uploaded.
+    spreadsheet_id: Google spreadsheet id.
+  """
+  # Changing directory to comply with "cred.json" path in "gsheet.py".
+  os.chdir('..')
+  exit_code = 0
+  if spreadsheet_id == "":
```

Comment:
Will it be the case? Aren't we confirming while running python file only?

Reply:
spreadsheet_id is optional argument ,hence , required.